src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_prompty.py

```diff
@@ -251,3 +256,28 @@ def test_prompty_trace(self, pf: PFClient):
         events = [pf.traces.get_event(item["attributes"]["event.id"]) for item in prompty_span.events]
         assert any(["prompt.template" in event["attributes"]["payload"] for event in events])
         assert any(["prompt.variables" in event["attributes"]["payload"] for event in events])
+
+    def test_prompty_with_sample(self, pf: PFClient):
+        prompty = Prompty.load(source=f"{PROMPTY_DIR}/prompty_example_with_sample.prompty")
```

Comment:
do we support load_flow?
Is it suggested way to import Prompty using
from promptflow.core._flow import AsyncPrompty, Prompty ?

Reply:
also add some simple cli test.

Reply:
The current recommended usage is, Flow.load(source="path/to/prompty.prompty")

Reply:
Add cli test cases.

Reply:
we will support load_flow in future